modules/installing-python-packages-on-your-notebook-server-in-code-server.adoc

```diff
@@ -0,0 +1,70 @@
+:_module-type: PROCEDURE
+
+[id="installing-python-packages-on-your-notebook-server-in-code-server_{context}"]
+= Installing Python packages on your notebook server in code-server
+
+[role='_abstract']
+You can install Python packages that are not part of the default notebook server by adding the package and the version to a `requirements.txt` file and then running the `pip install` command in a terminal window.
+
+ifndef::upstream[]
+NOTE: You can also install packages directly, but {org-name} recommends using a `requirements.txt` file so that the packages stated in the file can be easily re-used across different notebooks. In addition, using a `requirements.txt` file is also useful when using a S2I build to deploy a model.
+endif::[]
+ifdef::upstream[]
+NOTE: You can also install packages directly, but using a `requirements.txt` file so that the packages stated in the file can be easily re-used across different notebooks is recommended. In addition, using a `requirements.txt` file is also useful when using a S2I build to deploy a model.
+endif::[]
+
+.Prerequisites
+* A launched and running code-server notebook server.
+
+.Procedure
+. In your code-server workbench, from the Activity Bar, select the menu icon (image:images/codeserver-menu-icon.png[The Menu icon]) > *File* > *New Text File*  to create a new text file.
```

Comment:
```suggestion
. In your code-server workbench, from the Activity Bar, select the menu icon (image:images/codeserver-menu-icon.png[The Menu icon]) > *File* > *New Text File* to create a new text file.
```

does the double-space make a difference, actually? maybe the rendered docs is the same with single or double space?